markdown/articles/2020/12/20/historyofdoomsday.md

```diff
@@ -0,0 +1,1030 @@
+Dark Ritual and an LED, you could net mana each loop. To close the loop you could Intuition for 3 copies
+of Tendrils of Agony to win. It was very consistent boasting 8 maindeck tutor effects and maindeck removal like
+{{Chain of Vapor}} or {{Cabal Pit}}.
+
+It's here in our story where we re-focus on Emidln. Emidln had gotten a taste of competetive magic and so 
+started browsing forums to find out more information. Places like the Starcitygames forums, the mana drain
+forums (in the new legacy section, the mana leak) and the mtgsalvation forums. He started to meet and 
+become friends with people online, chatting via Aim, MSN, IRC, ICQ and a whole host of other messaging platforms.
+He realised he very much enjoyed the idea and style of the combo decks that were coming out and was invited
+to a specialised team forums for 'Team Blitzkrieg' or BZK for short. On the BZK forums all mana of combo decks
+were being discussed including early Spanish Inquisition lists (SI), the aforementioned Iggy Pop and some new
+decks that would start to evolve into more Modern storm lists and Emidln was in the middle of them.
+
+Spanish Inquistion is a deck that a lot of people still hold very dear to their heart to this day. The idea of 
+the deck is to utilise fast mana and ritual effects combined with 'draw 4's' like {{Cruel Bargain)) and
+{{Infernal Contract}}. {{Land Grant}}, along with {{Culling the Weak}} and creatures like {{Phyrexian Walker}}
+were used to cheat on mana and then the deck finished off opponents with either Tendrils of Agony or
+{{Goblin Charbelcher}}. There was a point in time where the SI decks tried a number of cards in them to 
+increase the *business* count available as the draw 4's could be unreliable. Cards like {{Diablic Intent}} worked
+alongside the 'tallmen' approach with {{Shield Sphere}} and friends however this did stretch reliance on the creature
+aspect. Doomsday was explored by Emidln and BZK friends; Evenpence and Desolutionist as a way to incorporate additional 
+business into the shell to some success but the deck, although fast, did not achieve any top tier status.
+
+<row variant="pile">{{!Cruel Bargain}} {{!LP}} {{!DR}} {{!DR}} {{!ToA}}</row>
+
+IN 2006 Iggy pop got a large boost in the form of {{Leyline of the Void}} from Guildpact which allowed it to mitigate the
+balanced nature of Ill-Gotten Gains by removing the opposing hand from the game entirely. It could also just be
+used to make an opponent go hellbent from turn 1. A very key card was printed in Dissession in the form of 
+{{Infernal Tutor}}. This card allowed for a combo with Lion's Eye Diamond to achiveve hellbent status and 
+effectively become a {{Demonic Tutor}} in doing so. It also gave rise to other branch decks like Nausea which ran
+{{Spoils of the Vault}}, {{Chromatic Sphere}} and Helm of Awakening. The deck actually ran a single copy of
```

Comment:
I think Nausea actually predated Guildpact. It was a port of Vintage SX ("Storm 10") and tried to use Helm of Awakening, Spheres, and Eggs + 4 Tendrils and Spoils for Tendrils/Helm.